Packages/manifest.json

```diff
@@ -28,7 +28,7 @@
     "com.unity.xr.oculus": "3.3.0",
     "com.unity.xr.openxr": "1.8.2",
     "com.zappar.xr.zapbox": "https://github.com/zappar-xr/zapbox-xr-sdk.git#3296cbf5046369801027a821fe9ff6082431a605",
-    "org.khronos.unitygltf": "https://github.com/icosa-mirror/UnityGLTF.git#dev",
+    "org.khronos.unitygltf": "https://github.com/icosa-mirror/UnityGLTF.git",
```

Comment:
What's different in our mirror?
Can we pin to a specific version? (e.g. `2.0.0-openbrush`, see meta sdk mirror for example)

Reply:
It was really future proofing as I expected to need to make some customisations. I'll consider changing this but I've already started deleting caches in prep for merging this so it will have to be a separate PR